pages/common/cmatrix.md

```diff
@@ -0,0 +1,20 @@
+# cmatrix
+
+> Shows a scrolling Matrix like screen in the terminal.
+> More information: <https://github.com/abishekvashok/cmatrix>.
+
+- Enable asynchronous scrolling:
+
+`cmatrix -a`
+
+- Display red text:
+
+`cmatrix -C {{red}}`
+
+- Enable rainbow mode:
+
+`cmatrix -r`
+
+- Set screen update delay to 2:
```

Comment:
2 what, seconds?

Reply:
The doc does not say but if I understand [the code](https://github.com/abishekvashok/cmatrix/blob/a2b78a109fc4b46d9a84ab556c5a96d2b0390a78/cmatrix.c#L856) correctly, I think it is centiseconds.